### DISCUSSION TITLE
How to run against a local build using `yarn link`?

### DISCUSSION THREAD

**Discussion by DanSchoppe** [visgl/deck.gl#7769] [Q&A] [answered]
Hello,

I'd like to point a local React project to a local build of `deck.gl`, ideally with hot reloading. Hopefully this is possible with `yarn link`. I've been through the [Contributing to deck.gl](https://github.com/visgl/deck.gl/blob/master/CONTRIBUTING.md#contributing-to-deckgl) documentation and gotten the project running locally (`yarn start-local` for the built-in examples). It's just not clear to me whether I can run a separate local project against my local `deck.gl` build.

Is there a way to `yarn link` with hot reloading? How about `yarn link` with a build step (not hot reloading)?

Thanks!

**Comment by Pessimistress** [comment, marked as answer]
There is no easy way to `yarn link` with the many sub modules. You may also run into issues if the local dependencies conflict with the dependencies installed in the deck repo.

For dev testing I usually modify one of the examples and run it with `yarn start-local`. Then just not commit the change in the example.
